addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -170,6 +170,22 @@ $o-wslides-fs-side-width: 300px;
     @include media-breakpoint-up(md) {
         border-top-width: 0;
     }
+
+    .o_wslides_js_channel_unsubscribe {
+        & > .fa-times {
+            display: none;
+        }
+    }
+
+    .o_wslides_js_channel_unsubscribe:hover {
```

Comment:
You factorize a little:

```
.o_wslides_js_channel_unsubscribe {
    > .fa-times {
        display: none;
    }
    
    &:hover {
        > .fa-check {
            display: none;
        }

        > .fa-times {
            display: inline-block;
        }
    }
}
```

Also, wits scss, I don't think you jave to specify the `&` every time.

Give it a try.